	The Posix manpage says strcasecmp() is declared in <strings.h>
	and this indeed required for AIX 4.3 and xlc.

diff -ur -N xpdf-3.03.orig/xpdf/GlobalParams.cc xpdf-3.03/xpdf/GlobalParams.cc
--- xpdf-3.03.orig/xpdf/GlobalParams.cc	2012-10-19 10:01:03.000000000 +0200
+++ xpdf-3.03/xpdf/GlobalParams.cc	2012-10-19 10:06:40.000000000 +0200
@@ -48,6 +48,8 @@
 
 #ifdef WIN32
 #  define strcasecmp stricmp
+#else
+#  include <strings.h>
 #endif
 
 #if MULTITHREADED
